decentraland/unity-explorer
# Fix: fixed interactability of floating panel in navmap on search resu…

## What does this PR change?

Fix #885
Fixes interactability of floating panel in navmap after clicking on a search result

...

## How to test the changes?



1. Launch the explorer
2. Use the navmap
3. Search a place
4. Click on it
5. Verify the floating panel is interactable correctly

## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
